README.md

```diff
@@ -0,0 +1,46 @@
+# Events Android
+
+The new Grinnell Events App created by the Android Development Team @ AppDev.
+
+####### Collaborators:
```

Comment:
If you use fewer `#`s then you can make this a header, not sure which you would prefer.

Reply:
My bad, should have been just two `#`s for the sub header.